exercises/practice/alphametics/.meta/config.json

```diff
@@ -26,5 +26,6 @@
     "example": [
       ".meta/example.py"
     ]
-  }
+  },
+  "test_runner": "false"
```

Comment:
I **think** this needs to be a boolean, not a string.

Reply:
```suggestion
  "test_runner": false
```